components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -1135,6 +1135,11 @@ def report_settings(module):
 # FEEDBACK_APP: 6 = OMERO.web
 FEEDBACK_APP = 6
 
+# For any given release of webgateway api, we may support
+# one or more versions of the api.
+# E.g. webgateway/api/v1.0/
+WEBGATEWAY_API_VERSIONS = ['1.0']
```

Comment:
We need to discuss our versioning strategy - this is just a placeholder for now. E.g. github just has 'v3' https://developer.github.com/v3/versions/